docs/understand/enterprise/orchestration.md

```diff
@@ -0,0 +1,389 @@
+
+With every new release of lakeFS or Fluffy, the lakeFS team releases a new [lakeFS 
+helm chart](https://artifacthub.io/packages/helm/lakefs/lakefs). Together with the granted
+Fluffy token, you can run the full lakeFS Enterprise solution.
+
+As an example, the following `values` file will run lakeFS Enterprise with OIDC integration.
+
+```yaml
+lakefsConfig: | 
+  logging:
+      level: "INFO"
+  blockstore:
+    type: s3
+  database:
+    type: postgres
+    postgres:
+      connection_string: <postgres-connection-string>
+  auth:
+    oidc:
+      # the claim that's provided by the OIDC provider (e.g Okta) that will be used as the username according to OIDC provider claims provided after successful authentication
+      friendly_name_claim_name: "<some-oidc-provider-claim-name>"
+      default_initial_groups: ["Developers"]
+    ui_config:
+      login_cookie_names:
+        - internal_auth_session
+        - oidc_auth_session
+ingress:
+  enabled: true
+  ingressClassName: <class-name>
+  hosts:
+    # the ingress that will be created for lakeFS
+    - host: <lakefs.ingress.domain>
+      paths: 
+       - /
+
+##################################################
+########### lakeFS enterprise - FLUFFY ###########
+##################################################
+
+fluffy:
+  enabled: true
+  image:
+    repository: treeverse/fluffy
+    tag: '0.4.0'
+    pullPolicy: IfNotPresent
+    privateRegistry:
+      enabled: true
+      secretToken: <dockerhub-token-fluffy-image>
+  fluffyConfig: |
+    logging:
+      format: "json"
+      level: "INFO"
+    database:
+      type: postgres
+      postgres:
+        connection_string: <postgres-connection-string>
+    auth:
+      serve_listen_address: 0.0.0.0:9000
+      logout_redirect_url: https://oidc-provider-url.com/logout/example
+      oidc:
+        enabled: true
+        url: https://oidc-provider-url.com/
+        client_id: <oidc-client-id>
+        callback_base_url: https://<lakefs.ingress.domain>
+        is_default_login: true
+        # the claim name that represents the client identifier in the OIDC provider (e.g Okta)
+        logout_client_id_query_parameter: client_id
+        # the query parameters that will be used to redirect the user to the OIDC provider (e.g Okta) after logout
+        logout_endpoint_query_parameters:
+          - returnTo
+          - https://<lakefs.ingress.domain>/oidc/login
+  secrets:
+    create: true
+  sso:
+    enabled: true
+    oidc:
+      enabled: true
+      # secret given by the OIDC provider (e.g auth0, Okta, etc)
+      client_secret: <oidc-client-secret>
+  rbac:
+    enabled: true
+
+useDevPostgres: true
+```
+
+1. The example uses OIDC authentication method. For other methods, see [SSO]({% link reference/security/sso.md %}).
+2. Database configuration must be identical between lakeFS and Fluffy.
+3. `useDevPostgres` isn't suitable for production. When used, a local dev postgres is created.
+
+
+## Using docker compose
```

Comment:
I think you should remove this section completely and even insist on it. 
Don't get me wrong - It's well written and useful **but** in reality I think we will have hard time maintaining this.
We can hardly keep up with the Helm examples being in Sync. 
Even if remembering that this section exists, testing Docker compose before updating the docs will impose too much friction. 

As an alternative minimal approach I would add a highlighted section "TIP" near Helm saying that to extract the final configuration in for K8S manifests without deploying the specific chart one can use `helm template` sub command to generate the manifests.

Reply:
Synced offline, decided to keep as is